I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 5, in script
    init python:
  File "game/script.rpy", line 23, in <module>
    Level("tutorial", _("Tutorial"))
TypeError: __init__() missing 1 required positional argument: 'unlocked'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 5, in script
    init python:
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/anastasiakozlova/renpy-8.1.3-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 23, in <module>
    Level("tutorial", _("Tutorial"))
TypeError: __init__() missing 1 required positional argument: 'unlocked'

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.1.3.23091805
Social Engineering Game 1.0
Thu Jan 11 12:02:11 2024
